### PR TITLE
bugfix: fix the nonetype bug from adding type

### PR DESCRIPTION
**Pull request by lanzhenw**
## What changes are proposed in this pull request?

(Please fill in changes proposed in this fix)

## How is this patch tested? If it is not, please explain why.

(Details)

### Is this a user-facing change that should be mentioned in the release notes?

-   [ ] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

File: fiftyone/server/query.py [hunk 1]
```diff
@@ -478,9 +478,12 @@ def run():
...
 
         for brain_method in data.brain_methods:
-            setattr(
-                brain_method.config, "type", brain_method.config.type().value
+            value = (
+                brain_method.config.type().value
+                if brain_method.config.type() is not None
+                else None
             )
+            setattr(brain_method.config, "type", value)
```
Comment (kaixi-wang) [inline fiftyone/server/query.py:486]: I'm a little confused here. You're reading the value from brain_method and setting the same value on brain method?

Comment (lanzhenw) [reply fiftyone/server/query.py:486]: We added the `type` property in the graphQL on query.py. Brain_method.config does not have`type`. 
When the user submits the similarity run, `/sort` route would refresh the dataset with `serialize_dataset`. And brain_method.config does not have `type`. That's why we are adding `type` to `brain_method.config` here.

Comment (benjaminpkane) [inline fiftyone/server/query.py:486]:
```suggestion
            value = brain_method.config.type()
            setattr(brain_method.config, "type", value.value if value else None)
```

**Review by benjaminpkane — APPROVED** [review]
LGTM!

**Pull request merged**